Demo/Fullscreen/InlineFilterDemoViewController.swift

```diff
@@ -2,82 +2,46 @@
 //  Copyright © FINN.no AS, Inc. All rights reserved.
 //
 
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
```

Comment:
How many copyright notices do we need in this file?  😂

Reply:
Haha 😄